packages/server/api/controllers/teamMembers.controller.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable */
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+// Get all team members
+const getAllTeamMembers = async () => {
+  try {
+    return await knex('TeamMembers').select('*');
```

Comment:
For this project, this is perfect, but in real life we would need to check if the user who is asking for the team members have accessrights 😄